### PR TITLE
Fix mAng3_c copy assignment, clean up list node templates

### PR DESCRIPTION
**Pull request by robojumper**
I *think* this way the subclasses that can be observed in dtors with multiple null checks actually make sense

### CODE REVIEW

File: include/d/a/d_a_t_wood_area.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    static const f32 scaleX;
+    static const f32 scaleY;
+
+private:
+    STATE_MGR_DECLARE(dAcTWoodArea_c);
+
+    dAcRef_c<dAcObjBase_c> mRefs[8];
+};
+
+// {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
```
Comment (lepelog) [inline include/d/a/d_a_t_wood_area.h:46]: ?

Comment (robojumper) [reply include/d/a/d_a_t_wood_area.h:46]: Huh, not sure what Zeldex wanted to do with these

Comment (lepelog) [reply include/d/a/d_a_t_wood_area.h:46]: Can probably just be removed then, looks good otherwise

Comment (elijah-thomas774) [reply include/d/a/d_a_t_wood_area.h:46]: Whoops sorry. I think was testing  cases with the particle indexing. Looks like I forgor the comment. It’s good to remove.

**Pull request merged**